### PR TITLE
IIIF2 comparison

### PR DESCRIPTION
**Pull request by donaldgray**
Write python script to compare differences between new and current P2 manifests, allowing for known differences.

Includes various small changes to P2 generation to handle any issues uncovered.

### CODE REVIEW

File: src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/LicencesAndRights/LegacyConfig/PlayerConfigProvider.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ private SimpleDialogue GetHelpDialogue()
...
             dict["ARR"] = "The work has been made available under an \"all rights reserved licence\". See the full catalogue record for further information about what rights you have to make copies of this work.";
             dict["Copyright not cleared"] = "The copyright of this item has not been evaluated. Please refer to the original publisher/creator of this item for more information. You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to your use. <br/>See <a target=\"_top\" href=\"http://rightsstatements.org/page/CNE/1.0/?language=en\">rightsstatements.org</a> for more information.";
-            dict["In copyright"] = "Conditions of use: it is possible this item is protected by copyright and/or related rights. You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to your use. For other uses you need to obtain permission from the rights-holder(s).";
+            dict[Constants.InCopyrightCondition] = Constants.InCopyrightConditionStatement;
```
Comment (tomcrane) [inline src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/LicencesAndRights/LegacyConfig/PlayerConfigProvider.cs:121]: I suspect you;ll find a few where you'll need to do exactly the same with the "Copyright not cleared" above this one.

File: src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/ProviderExtensions.cs [hunk 2]
```diff
@@ -13,19 +13,19 @@ public static void AddWellcomeProvider(this ResourceBase iiifResource, string ho
...
             {
-                Id = "https://wellcomecollection.org",
+                Id = Constants.WellcomeCollectionUri,
                 Label = Lang.Map("en",
-                    "Wellcome Collection",
+                    Constants.WellcomeCollection,
                     "183 Euston Road",
                     "London NW1 2BE UK",
                     "T +44(0)2076118722",
                     "E library@wellcome.ac.uk",
```
Comment (tomcrane) [inline src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/ProviderExtensions.cs:22]: This should be library@wellcomecollection.org
(from https://wellcomelibrary.org/using-the-library/services-and-facilities/contact-us/)
and spaces in the telephone number 
+44 (0)20 7611 8722

**Review by tomcrane — APPROVED** [review]

File: src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/LicencesAndRights/LegacyConfig/PlayerConfigProvider.cs (hunk 1 above)
Comment (donaldgray) [reply src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/LicencesAndRights/LegacyConfig/PlayerConfigProvider.cs]: Added handling for "Copyright not cleared"

File: src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/ProviderExtensions.cs (hunk 2 above)
Comment (donaldgray) [reply src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/ProviderExtensions.cs]: Updated contact details.

**Pull request merged**